src/enr/enr.ts

```diff
@@ -350,6 +350,6 @@ export class ENR extends Map<ENRKey, ENRValue> {
     return encoded;
   }
   encodeTxt(privateKey?: Buffer): string {
```

Comment:
I'd prefer to remove `Buffer` entirely. Iirc, the only reason we needed it (vs vanilla `Uint8Array`) was bc of bcrypto.
Maybe we can just do this as a separate PR (it's also a breaking change)

Reply:
That feels like a pretty big lift to include in what is already not a small PR.  Are you good with differing that to if/when we get this one merged?